src/Controller/ClassController.php

```diff
@@ -123,7 +123,7 @@ private function considerClassificationStoreForColumnConfig(Request $request, ?D
         if ($displayMode == ColumnConfigDisplayMode::DATA_OBJECT || $displayMode == ColumnConfigDisplayMode::RELEVANT) {
             $targetObjectId = $request->get('target_oid');
 
-            if (($targetObject = DataObject\Concrete::getById($targetObjectId)) && !$targetObject instanceof DataObject\Folder) {
```

Comment:
why removed `DataObject\Folder` check?

Reply:
Concrete can not be a Folder, right?

Reply:
why not? It calls the `AbstractObject::getById`

Reply:
The getById methods of Pimcore check the static type. If this not match it returns null.

See https://github.com/pimcore/pimcore/blob/1f5e6be5d32eec3c611cd37807d2c6f1396d8f55/models/DataObject/AbstractObject.php#L422-L424

Reply:
I see that Concrete is excluded from this match...
https://github.com/pimcore/pimcore/blob/1f5e6be5d32eec3c611cd37807d2c6f1396d8f55/models/DataObject/AbstractObject.php#L519
Why? This makes no sense and make the return type static|null invalid. Can we change this in the core?

Reply:
On v11, we don't return folder objects so we are fine with the change.